eventmesh-examples/build.gradle

```diff
@@ -17,15 +17,14 @@
 
 def grpcVersion = '1.43.2'
 
-configurations {
-    implementation.exclude group: 'org.springframework.boot', module: 'spring-boot-starter-logging'
-}
-
 dependencies {
     implementation project(":eventmesh-sdks:eventmesh-sdk-java")
     implementation project(":eventmesh-common")
     implementation project(":eventmesh-storage-plugin:eventmesh-storage-api")
     implementation project(":eventmesh-connectors:eventmesh-connector-spring")
+    implementation('org.springframework.boot:spring-boot-starter') {
+        exclude module: 'spring-boot-starter-logging'
+    }
     implementation 'org.springframework.boot:spring-boot-starter-web'
```

Comment:
`logback` was introduced back by `spring-boot-starter-web`. I will remove it completely in https://github.com/apache/eventmesh/pull/4831/commits/be06ef7441d58b6cf148ca6fe7457525502fcf4d and https://github.com/apache/eventmesh/pull/4827/commits/0261ef6174d0a431074226b96589c375b4f11242.